### ISSUE TITLE
[BUG] Linux: Browser mouse coordinates wrong on scaled displays

### ISSUE THREAD

**Issue by christianhujer** [obsproject/obs-studio#2615]
## Platform

Operating system and version: Kubuntu 18.04.04 LTS
Details:
```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.4 LTS
Release:        18.04
Codename:       bionic
$ uname -a
Linux Nelkinda-Blade-Stealth-2 4.15.0-91-generic #92-Ubuntu SMP Fri Feb 28 11:09:48 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

OBS Studio version: 25.0.3 (linux)
Installed via: Ubuntu installation instructions
```
$ cat /etc/apt/sources.list.d/obsproject-ubuntu-obs-studio-bionic.list
deb http://ppa.launchpad.net/obsproject/obs-studio/ubuntu bionic main
# deb-src http://ppa.launchpad.net/obsproject/obs-studio/ubuntu bionic main
```

## Expected Behavior
When using "Interact" to interact with the browser, the mouse coordinates for the user and the mouse coordinates for the browser match.

## Current Behavior
At least on systems with a scaled display (in my case, a 4K screen with a display scale of 2), the mouse coordinates reported to the browser are wrong. They seem wrong by a factor identical to the display scale.

## Steps to Reproduce
Live example: https://youtu.be/CSIverVzRUc

1. Configure your system to have a display scale. You might need to restart X11 or SDDM (usually logging out and in again is sufficient) to ensure that this is in effect.
2. Add a Browser to the scene.
3. Configure the browser to display a page with elements with which you want to interact with using the mouse.
4. Interact with the Browser.
5. Try to hit any of the elements on the page with the mouse.

## Additional information
In case this is relevant:
Desktop: KDE Plasma

## Workaround
Use a fullscreen browser on another screen and use a screen source instead. (Only works well for people with multiple screens.)

**Comment by scorpi0on** [comment]
mac os 10.15 also has same problem with browser source

**Comment by cujomalainey** [comment]
On OSX you also can't type certain characters (like a `:`)

**Comment by WizardCM** [comment]
Fixed in https://github.com/obsproject/obs-studio/pull/3833 - available in 26.1.0. It may take a few days to reach your distro.

**Comment by christianhujer** [comment]
Thank you so much! Now just waiting for OBS 26.1.0 becoming available for Ubuntu 20.04 LTS.
